### ISSUE TITLE
error on mobile devices

### ISSUE THREAD

**Issue by clancyGruver** [tensorflow/tfjs#5301]
**System information**
- OS Platform and Distribution: Android 11
- Mobile device : Samsung galaxy S10, iphone 7
- TensorFlow.js installed from npm:
- TensorFlow.js 3.7.0:
- tfjs-tflite.js 0.0.1-alpha.4:
- Browser version: Chrome 91

**Describe the current behavior**
- tflite_web_api_cc_simd.js:9 Uncaught (in promise) RuntimeError: abort(undefined). Build with -s ASSERTIONS=1 for more info.
    at abort (tflite_web_api_cc_simd.js:9)
    at _abort (tflite_web_api_cc_simd.js:9)
    at tflite_web_api_cc_simd.wasm:0x188bf
    at tflite_web_api_cc_simd.wasm:0x1806fa
    at tflite_web_api_cc_simd.wasm:0x181025
    at tflite_web_api_cc_simd.wasm:0x181123
    at tflite_web_api_cc_simd.wasm:0x26afd3
    at tflite_web_api_cc_simd.wasm:0x28262f
    at tflite_web_api_cc_simd.wasm:0x28268b
    at tflite_web_api_cc_simd.wasm:0x33555

**Standalone code to reproduce the issue**
import { loadTFLiteModel, TFLiteModel, setWasmPath, getWasmFeatures } from '@tensorflow/tfjs-tflite';
setWasmPath('tflite-wasms/');

**Comment by jinjingforever** [comment]
Hi @clancyGruver,

Thanks for the report! I tried the following code on my Android 11 emulator with Chrome 91 (SIMD enabled) and it seems to be working fine ([screenshot](https://drive.google.com/file/d/1HRrbWqP9Dx83ddEeCQ0XJa7aKrKi_Gmz/view?usp=sharing)). Maybe there was more code after `setWasmPath` you didn't post? (looks like the error occurred when you were trying to load a model?).

```js
// Import @tensorflow/tfjs-tflite.
import {getWasmFeatures, loadTFLiteModel, setWasmPath} from '@tensorflow/tfjs-tflite';

setWasmPath('https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-tflite/dist/');

async function start() {
  console.log('start');
  const wasmFeatures = await getWasmFeatures();
  console.log(
      'simd :' + wasmFeatures.simd +
      ', multi-threading: ' + wasmFeatures.multiThreading);

  // Load model runner with the cartoonizer tflite model.
  await loadTFLiteModel(
      'https://tfhub.dev/sayakpaul/lite-model/cartoongan/fp16/1',
  );
  console.log('model loaded');
}

start();
```

**Comment by clancyGruver** [comment]
An emulator isn't the best solution for testing phones, because you still use your processor instead of phone's one.
Here is a console screenshot for a mobile phone. It's Galaxy S10.
The same code works perfect on desktop.
![image](https://user-images.githubusercontent.com/14860440/125576570-22ef4c0e-c0e2-4f46-b2f4-2861adc63935.png)

**Comment by jinjingforever** [comment]
Thank you! You are right about the emulator. I will try to find an Android device to test this (works fine on my iPhone). From the error message, it looks like something inside the TFLite runtime. I will ask the team about this and report back if I get anything. Thanks!

**Comment by clancyGruver** [comment]
Thank! It will be great.

**Comment by jinjingforever** [comment]
I was able to use the browserstack's Real Device Cloud to run the same code on Galaxy S10 (with Android 9 and Chrome 89), but I didn't see the error you saw. I also tried Galaxy S21 with Android 11 and Chrome 91 and it seems to be working fine as well. 

Is it possible that the model is not downloaded successfully from tfhub? (or are you using a different tflite model?) Thanks!

![Screen Shot 2021-07-15 at 11 39 06 AM](https://user-images.githubusercontent.com/8752427/125840348-96d0603f-63e4-4459-9a96-c9fd889a8d28.png)
![Screen Shot 2021-07-15 at 11 33 52 AM](https://user-images.githubusercontent.com/8752427/125840361-9304bb2c-830c-4016-bd74-b19b9d0e625e.png)
